fe2-android/README.md

```diff
@@ -0,0 +1,67 @@
+# student20_pop: fe2-android branch
+Proof-of-personhood, fall 2020: Android native front-end
```

Comment:
please update references to Spring 2021 / student_21_pop , remove notion of "branch"